src/styles/blocks/stars.scss

```
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star-active.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+  }
+
+  &__star:last-child {
+    background-image: url(../images/star.svg);
+
+  }
```

can you do it with `@for` scss loop?
```suggestion
  &__star:last-child {
    background-image: url(../images/star.svg);
  }
```